File: 136_Single_Number/Singel_Number.go
func singleNumber(nums []int) int {
    arr_size:=len(nums)
    sort.Ints(nums)
    for k, v := range nums {
        if k > 0 && k % 2 != 0 {
            if v != nums[k-1] {
                return nums[k-1]
            }
        }  
    }
    return nums[arr_size-1]
}

File: 136_Single_Number/Singel_Number.py
class Solution(object):
    def singleNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        nums.sort()
        for i in range(0,len(nums),2):
            if i+1 >= len(nums): return nums[i]
            elif nums[i] != nums[i+1]: return nums[i]
